main.py

```diff
@@ -0,0 +1,69 @@
+from fastapi import FastAPI, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+import crud
+import schemas
+
+from db.database import SessionLocal
+
+app = FastAPI()
+
+
+def get_db():
+    db = SessionLocal()
+
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+@app.post("/authors/", response_model=schemas.Author)
+def create_author(
+        author: schemas.AuthorCreate,
+        db: Session = Depends(get_db)
+):
+    return crud.create_author(db=db, author=author)
+
+
+@app.get("/authors/", response_model=list[schemas.Author])
+def get_authors(
+        skip: int = 0,
+        limit: int = 10,
+        db: Session = Depends(get_db),
+):
+    return crud.get_authors(db=db, skip=skip, limit=limit)
+
+
+@app.get("/authors/{id}/", response_model=schemas.Author)
+def get_author_by_id(
+        author_id: int,
```

Comment:
The parameter name `author_id` in the `get_author_by_id` function should match the path parameter `{id}` in the route. Change `author_id` to `id` to ensure the path parameter is correctly captured.